src/syscall/syscall_unix.go

```diff
@@ -1,3 +1,6 @@
+//go:build linux || unix
+// +build linux unix
```

Comment:
```suggestion
//go:build linux || unix
```

You can remove the old `// +build` lines.